src/qiboml/models/keras.py

```diff
@@ -15,7 +15,7 @@
 
 BACKEND_2_DIFFERENTIATION = {
     "pytorch": "PSR",
-    "tensorflow": None,
+    "tensorflow": "PSR",
```

Comment:
```suggestion
    "tensorflow": None,
```

I would say that by default `keras` interface will use the native `tf` differentiation rather than `PSR`. In any case, the automatic selection of the differentiation method is likely to be changed in #42